pegasus/forms/professional_development_workshop.rb

```diff
@@ -119,9 +119,12 @@ def self.solr_query(params)
       first_date_dt: '[NOW TO *]',
     }.map{|key,value| "#{key}:#{value}"}.join(' AND ')
 
+    fl = 'location_p,location_name_p,location_name_s,dates_ss,id'
```

Comment:
How did you decide which to whitelist? Like what about grade_level and (school) type?